tests/functional/z_cluster/nodes/test_node_replacement_proactive.py

```
@@ -345,25 +353,33 @@ class TestNodeReplacementTwice(ManageTest):
             ct_pod = pod.get_ceph_tools_pod()
             tree_output = ct_pod.exec_ceph_cmd(ceph_cmd="ceph osd tree")
-            log.info("ceph osd tree output:")
-            log.info(tree_output)
+            logger.info("ceph osd tree output:")
+            logger.info(tree_output)
 
+            logger.assertion(
+                f"Deleted node '{node_name_to_delete}' not in ceph osd tree"
+            )
             assert not (
                 node_name_to_delete in str(tree_output)
```

Ruff (0.15.15)

[error] Test for membership should be `not in`

Convert to `not in`

(E713)